tests/testsuite/package.rs

```diff
@@ -3638,30 +3638,30 @@ edition = "2015"
 name = "foo"
 version = "0.0.1"
 authors = []
-build = 'src/build.rs'
+build = "src/build.rs"
 description = "foo"
 documentation = "docs.rs/foo"
-readme = 'docs/README.md'
-license-file = 'docs/LICENSE'
+readme = "docs/README.md"
+license-file = "docs/LICENSE"
 
 [lib]
-path = 'src/lib.rs'
+path = "src/lib.rs"
```

Comment:
cargo was outputting `path = 'src\lib.rs'` but cargo-test-support converts `\` to `/`.

So the only hint that `\` was present is the use of single-quotes because `toml` prefers to avoid escaping by emitting string literals.